docs/content/preview/integrations/camunda.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Camunda
+linkTitle: Camunda
+description: Use Camunda with YSQL 
+aliases:
+section: INTEGRATIONS
+menu:
+  preview:
+    identifier: camunda
+    weight: 571
+isTocNested: true
+showAsideToc: true
+---
+
+[Camunda](https://camunda.com/) is a Java-based framework supporting BPMN for workflow and process automation, CMMN for Case Management and DMN for Business Decision Management. This page shows how it can be integrated with YugabyteDB's YSQL API.
+
+## Prerequisites
+
+Before you start using the KairosDB plugin, ensure that you have:
+Make sure you have the following set of tools installed:
+
+- Java JDK 1.8+,
+- A modern web browser (recent Firefox, Chrome or Microsoft Edge will work fine).
+- Eclipse integrated development environment (IDE)/Visual Studio Code.
+- A YugabyteDB cluster. Refer to [YugabyteDB Quick start guide](/preview/quick-start/) to install and start a local cluster.
+- YugabyteDB JDBC Driver jar [file](https://repo1.maven.org/maven2/com/yugabyte/jdbc-yugabytedb/42.3.5-yb-1/jdbc-yugabytedb-42.3.5-yb-1.jar).
+- Postgres JDBC Driver jar [file](https://jdbc.postgresql.org/download/postgresql-42.3.5.jar).
+- Camunda Modeler, you can get it from [here](https://camunda.com/download/modeler).
+- NodeJS >= v10 (Download available [here](https://nodejs.org/en/download/)).
+- [Postman API Platform](https://www.postman.com/downloads/) (optional).
+
+## Configure Camunda Platform 7
+- Download Camunda Platform 7 from [here](https://camunda.com/download/) and unzip it.
+- Configuration the following files
```

Comment:
Wording is not quite clear. It could be something like 'In following configuration files, find the spring.datasource stanza and edit as shown below.'